eq3bt/eq3cli.py

```diff
@@ -44,57 +45,75 @@ def cli(ctx, mac, interface, debug):
 @pass_dev
 def temp(dev, target):
     """ Gets or sets the target temperature."""
-    click.echo("Current target temp: %s" % dev.target_temperature)
-    if target:
-        click.echo("Setting target temp: %s" % target)
-        dev.target_temperature = target
+    if not dev.connection_error:
+        click.echo("Current target temp: %s" % dev.target_temperature)
+        if target:
+            click.echo("Setting target temp: %s" % target)
+            dev.target_temperature = target
+    else:
+        click.echo("Connection error")
```

Comment:
```suggestion
    if dev.connection_error:
        click.echo("Connection error")
        return
        
    click.echo("Current target temp: %s" % dev.target_temperature)
    if target:
        click.echo("Setting target temp: %s" % target)
        dev.target_temperature = target
```
It is better to return early for error cases, but see my commentary above regarding to how to handle errors in a nicer way with no need for adjusting each and every command separately.